packages/app/src/state/futures/reducer.ts

```diff
@@ -738,6 +744,36 @@ const futuresSlice = createSlice({
 		builder.addCase(fetchFundingRatesHistory.fulfilled, (futuresState, { payload }) => {
 			futuresState.historicalFundingRates[payload.marketAsset] = payload.rates
 		})
+
+		// Trading Fees by given epoch
+		builder.addCase(fetchFuturesFees.pending, (futuresState) => {
+			futuresState.queryStatuses.futuresFees = LOADING_STATUS
+		})
+		builder.addCase(fetchFuturesFees.fulfilled, (futuresState, action) => {
+			futuresState.queryStatuses.futuresFees = SUCCESS_STATUS
+			futuresState.futuresFees = action.payload.totalFuturesFeePaid
+		})
+		builder.addCase(fetchFuturesFees.rejected, (futuresState) => {
+			futuresState.queryStatuses.futuresFees = {
+				status: FetchStatus.Error,
+				error: 'Failed to fetch fee data',
+			}
+		})
+
+		// Trading Fees by given epoch and trader
+		builder.addCase(fetchFuturesFeesForAccount.pending, (futuresState) => {
+			futuresState.queryStatuses.futuresFees = LOADING_STATUS
```

Comment:
This is the same query status as above `futuresFees`, should probably use a new query status item for a separate action otherwise they would conflict.

Reply:
Good catch. I already defined a new query status but mistyped it here. Thanks for pointing it out.